packages/client/src/Dashboard.tsx

```diff
@@ -109,9 +111,23 @@ export default function Dashboard({ accountId, navigate }: Props) {
   const { data, error, loading } = graphql.useGetAccountQuery({
     variables: { accountId: accountId! }
   })
+
   const conversations = data && data.account && data.account.conversations
   const [selected, dispatch] = Sel.useSelectedConversations(conversations)
 
+  let isStarred = false
+
+  selected.forEach(select => {
+    conversations &&
+      conversations.forEach(conversation => {
+        if (conversation.id === select) {
+          if (conversation.isStarred) {
+            isStarred = true
+          }
+        }
+      })
+  })
```

Comment:
Instead of `forEach` would you use `reduce`? We can work on that change together if you want.